packages/survey-creator-react/package.json

```diff
@@ -21,7 +21,7 @@
     "doc:update": "chmod +x docupdate_npm.sh && ./docupdate_npm.sh",
     "testcafe": "concurrently \"http-server --silent\" \"testcafe -c 4 chrome ../../testCafe/ --selector-timeout 1500\"",
     "testcafe:ci": "http-server --silent & testcafe -c 4 -q chrome:headless ../../testCafe/ --selector-timeout 1500 --reporter minimal,dashboard",
-    "testcafe:file": "concurrently \"http-server --silent\" \"testcafe chrome ../../testCafe/property-grid/test.ts --reporter minimal --debug-mode\"",
+    "testcafe:file": "concurrently \"http-server --silent\" \"testcafe chrome ../../testCafe/designer/page-navigator.ts --reporter minimal --debug-mode\"",
```

Comment:
Maybe you should not give changes to package.json

Reply:
Why? It doesn't matter.